hadoop-hdds/docs/content/feature/SCM-HA.md

```diff
@@ -90,6 +90,85 @@ ozone scm --bootstrap
 
 Note: both commands perform one-time initialization.  SCM still needs to be started by running `ozone --daemon start scm`.
 
+## SCM Leader Transfer
+
+The `ozone admin scm transfer` command allows you to manually transfer the leadership of the Storage Container Manager (SCM) Raft group to a specific SCM node or to a randomly chosen follower.
+
+Be aware of the node's status(eg. Safemode, Operational status), ozone currently has no ability to check the target node's status before transfering the leadership.
+
+### Usage
+
+```bash
+ozone admin scm transfer -id <SCM_SERVICE_ID> -n <NEW_LEADER_ID>
+ozone admin scm transfer -id <SCM_SERVICE_ID> -r
+```
+
+*   `-id, --service-id`: Specifies the SCM Service ID.
+*   `-n, --newLeaderId, --new-leader-id`: The node ID of the SCM to which leadership will be transferred (e.g., `scm1`).
```

Comment:
From https://issues.apache.org/jira/browse/HDDS-11839, we cannot use the SCM Node ID as the --new-leader-id

cc: @peterxcli

Reply:
updated

Reply:
Thanks for the update. Please also help to update the following SCM transfer leader sections as well on how to get the SCM UUID using the `ozone admin scm roles` (the output should also include the SCM UUID).

Reply:
![image](https://github.com/user-attachments/assets/28aa4dfa-27d4-4450-8710-6cfd675378d2)

Ref: https://github.com/apache/ozone/pull/7278

So we might need to change the node ID format from scm1, scm2, scm3 to the UUID instead.

Reply:
@jojochuang Thanks for the update. Just need to update the `ozone admin scm roles --table` node ID to be UUID and we're good.